generate-release-notes.js

```diff
@@ -10,11 +10,12 @@ async function generateReleaseNotes({
 	exclude = '',
 	commitTemplate = '- {hash} {title}',
 	releaseTemplate = '{commits}\n\n{range}',
-	dateFormat = 'short'
+	dateFormat = 'short',
+	reverseSort = false
```

Comment:
I'd use ascending/descending instead of double negations (reverse = false)
```suggestion
	sort = 'desc'
```

Reply:
OK